php-files/functions.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @return PDO
+ * Connects to database
+ */
+function databaseConnect()
+{
+    $db = new PDO('mysql:host=192.168.20.20; dbname=fish-finder', 'root', '');
+    return $db;
+}
+
+/**
+ * @param $db
+ * @return array
+ * Queries the database to get multidimensional array of all data required to display.
+ */
+function getFish($db): array
+{
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $sql = $db->prepare('SELECT `fish`.`name`,`fish`.`species`,`fish`.`img-filepath`,`fish`.`length`,`fish`.`aggression`,`fish`.`color`,`fish`.`pattern` FROM `fish`;');
```

Comment:
If you only have one table you don't need to qualify every field name with the table name - `fish`.`name` could just as well be `name`, etc.